hunting-buddy.lic

```diff
@@ -259,6 +265,21 @@ class HuntingBuddy
         echo('***STATUS*** stopping due to time')
         break
       end
+      if Flags['familiar_drag'] && @bail_on_fam
+        #echo(Flags['familiar_drag'])
+        #echo("@bail_on_fam is set to #{@bail_on_fam}")
+        echo('***STATUS*** stopping due to familiar dragging from stun')
+        Flags.reset('familiar_drag')
+        break
+      end
+        #else
+      if Flags['familiar_drag'] && !@bail_on_fam
+          #echo(Flags['familiar_drag'])
+          #echo("@bail_on_fam is set to #{@bail_on_fam}")
+          echo('***STATUS*** heading back to room - familiar drug while stunned')
+          DRC.wait_for_script_to_complete('go2', [hunting_room])
```

Comment:
how about `DRC.walk_to(hunting_room)`?

Reply:
it sure does, unless the movement is over a stringproc using an fput for movement instead of move directly, which will also break go2

Reply:
the move command itself in lich is what does the retreating

Reply:
So, you want me to put DRC.walk_to(hunting_room) instead of the go2? Okay.